package-lock.json

```diff
@@ -54294,7 +54294,6 @@
 		"packages/base-styles": {
 			"name": "@wordpress/base-styles",
 			"version": "4.43.0",
-			"dev": true,
```

Comment:
Is there a reason for removing this?

Reply:
just a bad package-lock update, thanks for raising – it is now fixed